### ISSUE TITLE
About time between data and data

### ISSUE THREAD

**Issue by twyun24** [AIStream-Peelout/flow-forecast#500]
Hello sir, 
I got some issues during training 
your data "keag_small.csv"'s data time interval is an hour by hour.
But i got more quantities of data in USGS, the recent data comes out every 15minutes.
(just like 12:30 and next is 12:45 next is 13:00 ...etc) 

1)So is it OK to run directly in model da_rnn, or DO I have to change some specific columns?
(since time intervals are different between two type of data)

2)I'm just working on "learning rate", "batch_size", "epochs" in file da_rnn.json 
to minimize the cost of train_loss and validation_loss. 
What do you recommend to change first above "lr","batch_size","epochs"?
I wanna settle two of them down... and try different numbers on the rest element.
Waiting for your advice.

Thanks always.

**Comment by isaacmg** [comment]
A model trained on hourly data cannot predict on data coming in 15 minute intervals. If however you train the model on data in 15 minute intervals it should work. Also on separate  note that even though the USGS records data in 15 minute intervals it is only transmitted to the site once per hour. This takes away much of the advantage of it.
